src/Submission/ProjectSubmission.jsx

```diff
@@ -29,33 +30,28 @@ const ProjectSubmission = (props) => {
     }
     return <MyDataModelGraph project={props.project} />;
   };
-  const displaySubmissionUIComponents = (project, userAuthMapping) => {
-    if (
-      !useArboristUI
-      || (isRootUrl(project) && userHasSheepdogProgramAdmin(userAuthMapping))
-      || (isProgramUrl(project) && userHasSheepdogProjectAdmin(userAuthMapping))
-      || userHasMethodOnProject('create', project, userAuthMapping)
-      || userHasMethodOnProject('update', project, userAuthMapping)
-    ) {
-      return (
-        <React.Fragment>
-          <MySubmitForm />
-          <MySubmitTSV project={project} />
-        </React.Fragment>
-      );
-    }
-    return null;
-  };
 
   return (
-    <div className='project-submission'>
-      <h2 className='project-submission__title'>{props.project}</h2>
-      {
-        <Link className='project-submission__link' to={`/${props.project}/search`}>browse nodes</Link>
-      }
-      { displaySubmissionUIComponents(props.project, props.userAuthMapping) }
-      { displayData() }
-    </div>
+    shouldDisplaySubmissionUIComponents(
+      props.project,
+      props.userAuthMapping,
+      useArboristUI,
+    ) ?
+      <div className='project-submission'>
+        <h2 className='project-submission__title'>{props.project}</h2>
+        {
+          <Link className='project-submission__link' to={`/${props.project}/search`}>browse nodes</Link>
+        }
+        {
```

Comment:
here we need to check for `userHasMethodOnProject('create', project, userAuthMapping) || userHasMethodOnProject('update', project, userAuthMapping)` in order to determine whether we should render the submission component